### ISSUE TITLE
Bug: Cannot open window when click dock

### ISSUE THREAD

**Issue by zeroskylian** [heyman/heynote#238]
Is this a bug?
MacOS 14.5
App version 1.7.1

![录制于 2024-07-04 19 27 07](https://github.com/heyman/heynote/assets/7043428/7c0469c7-a952-483f-b11d-43d394ac467e)

**Comment by heyman** [comment]
Yes, this does sound like a bug. I'm not able to reproduce the issue.

Could you test with the latest beta version? How does the "General" page of Heynote's settings look?

**Comment by heyman** [comment]
Thanks! Managed to reproduce it. Should be fixed now.
